docs/CONNECTOR_PROTOCOL.md

```diff
@@ -258,11 +261,14 @@ In addition to the connector index `.elastic-connectors`, we have an additional
     }
   };
   index_name: string; -> The name of the content index
-  worker_hostname: string; -> The hostname of the worker to run the sync job
-  indexed_document_count: number; -> Number of documents indexed in the sync job
-  deleted_document_count: number; -> Number of documents deleted in the sync job
-  created_at: date; -> The date/time when the sync job is created
-  completed_at: date; -> The data/time when the sync job is completed
+  indexed_document_count: number; -> Number of documents indexed in the job
+  indexed_document_volume: number; -> The volume (in bytes) of documents indexed in the job
```

Comment:
Hi @sphilipse , do you need `last_indexed_document_volume` in `.elastic-connectors`?

Reply:
We don't have an immediate use for it.